scripts/eurostat/health_determinants/Alcohol_Consumption/download.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2022 Google LLC
```

Comment:
Can we have one single download.py module across Eurostat - Health Determinants - (PR700, 701, 702, 703)

Reply:
Done the change.